docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6
+MAINTAINER krev@triax.com
+
+# Tools
+RUN apt-get update && \
+    apt-get install -y ser2net
+
+WORKDIR /opt/labgrid
+
+# Ensure that labgrid is build from current dir
+COPY ./ ./
+RUN pip install crossbar
+RUN pip install -e /opt/labgrid
+RUN git clone https://github.com/vishnubob/wait-for-it.git /opt/wait-for-it
+
+RUN echo "stty rows 33 cols 204" >> /etc/bash.bashrc
```

Comment:
You are right this is strange. This is because the inhered images are configured for very small terminals, making interactive sessions cumbersome, but this is a hack that I will remove